### PR TITLE
Reduce likelyhood of random test failure.

### LINKED ISSUE #132: random tests fail occasionally
Some of the randomized tests in `tests/test-engine/racket-tests` fail occasionally. See http://drdr.racket-lang.org/56554/racket/share/pkgs/htdp-test/tests/test-engine/racket-tests.rkt for an example. 

The relevant failing tests are: 
- https://github.com/racket/htdp/blob/master/htdp-test/tests/test-engine/racket-tests.rkt#L229-L233
- https://github.com/racket/htdp/blob/master/htdp-test/tests/test-engine/racket-tests.rkt#L240-L241

I'm not certain what the specification being tested here is, or I would propose a fix.

The code was originally added in https://github.com/racket/htdp/commit/26e36b6dade7d6c7c95ea1c019e4a7808c82a8ed and may actually be by @dvanhorn.

### PR DESCRIPTION
**Pull request by samth**
Closes #132.

### CODE REVIEW

**Review by mfelleisen — APPROVED** [review]

File: htdp-test/tests/test-engine/racket-tests.rkt [hunk 1]
```diff
@@ -221,23 +221,25 @@
...
 
-(define (h _x) (car (list (random 50) (random 20))))
+(define (h _x) (car (list (random 50) (random 20) (random 100) (random 70))))
 
-(check-random (h 0) (begin0 (random 50) (random 20)))
+(check-random (h 0) (begin0 (random 50) (random 20) (random 100) (random 70)))
 (check-success)
 
-(check-random (h 0) (begin (random 20) (random 50))) ;; fails 
+(check-random (h 0) (begin (random 20) (random 50) (random 70) (random 100))) ;; fails
```
Comment (mfelleisen) [inline htdp-test/tests/test-engine/racket-tests.rkt:242]: @samth Do you want to keep this "fails" comment?

Comment (samth) [reply htdp-test/tests/test-engine/racket-tests.rkt:242]: I assumed that this comment meant that the test didn't pass and that's why it's checked for failing.

**Comment by mfelleisen** [comment]
I mistakenly assumed that you wanted to make it pass.

**Comment by samth** [comment]
It looks like a surprising number of the tests fail, so I'll investigate.

**Comment by samth** [comment]
I think the test failures here are just because the stepper tests continue to fail, but it's hard to tell because there's a lot of error printouts here.

**Pull request merged**